docs/source/format/CanonicalExtensions.rst

```diff
@@ -283,6 +283,28 @@ UUID
    A specific UUID version is not required or guaranteed. This extension represents
    UUIDs as FixedSizeBinary(16) with big-endian notation and does not interpret the bytes in any way.
 
+8-bit Boolean
+=============
+
+Bool8 represents a boolean value using 1 byte (8 bits) to store each value instead of only 1 bit as in
+the native Arrow Boolean type. Although less compact that the native representation, Bool8 may have
```

Comment:
"native" is a very overloaded term. Perhaps "original" is a better word choice here?

Reply:
typo: that -> than

Reply:
Thanks, fixed